torch/testing/_internal/common_nn.py

```diff
@@ -2940,11 +3263,18 @@ def padding3d_circular(input, pad):
     'CTCLoss': ctcloss_reference,
 }
 
+# NOTE: Instead of creating these weight tensors on the spot during
```

Comment:
Is this sufficient? Are there any modules that create random weights?

Reply:
Thanks for the catch - I actually don't think these constant tensors are necessary now, because we are always copying Python module's params / buffers to the C++ module. I have removed them :D